src/__tests__/jwe-vectors.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable */
 // it.skip('not a test', () => {})
 module.exports = {
-  dir: {
+  ecdhEsXc20p: {
```

Comment:
This is not correct. there is no key exchange involved when using direct encryption!
The protected header of the vector below states:
```json
{
  "alg": "dir",
  "enc": "XC20P"
}
```

Reply:
Right, will rename it to xc20p though because it is jwe-vector.js which i guess can expand on other content encryption algorithm than XC20p. IMO, it is nicer to group the test vectors by algorithm.

Reply:
makes sense 👍